completions/xdg-mime

```diff
@@ -2,9 +2,18 @@
 
 _xdg_mime_mimetype()
 {
-    COMPREPLY+=($(compgen -S / -W 'application audio font image message model
-                                    multipart text video' -- "$cur"))
-    [[ ${COMPREPLY-} == */ ]] && compopt -o nospace
+    local d i
```

Comment:
The change to `OLDPWD` can be localized (if we still perform `cd` in the main shell).

```suggestion
    local d i OLDPWD=$OLDPWD
```

Reply:
4fc5c89a462d22fdc685af79c8eade8ae476dba4 no longer `cd`s in the main shell, removed `ignore_env` from the test.